src/sentry/static/sentry/app/components/panels/panelBody.tsx

```diff
@@ -9,33 +9,34 @@ import textStyles from 'app/styles/text';
 type FlexComponentProps = Omit<React.ComponentPropsWithoutRef<typeof Flex>, 'theme'>;
 
 type Props = FlexComponentProps & {
-  disablePadding?: boolean;
-  direction?: FlexComponentProps['flexDirection'];
+  flexible?: boolean;
```

Comment:
What about `isFlex`

Reply:
`<PanelBody flexible withPadding>` vs `<PanelBody isFlexible withPadding>`. The former felt more "natural" to me but I'm not too vested in the name. We just cannot use `flex` as it clashes with existing props on `FlexComponentProps`.